dev/io.openliberty.jakarta.xmlBinding.4.0/bnd.bnd

```diff
@@ -47,4 +47,4 @@ publish.wlp.jar.suffix: dev/api/spec
 -buildpath: \
 	jakarta.xml.bind:jakarta.xml.bind-api;strategy=exact;version='4.0.0',\
 	io.openliberty.org.glassfish.hk2.osgi-resource-locator,\
-	com.ibm.ws.logging
+	com.ibm.ws.logging	
```

Comment:
I know that you probably did not indend to change this file under this PR but since you have...
a) Why does a Jakarta API bundle require our logging code to build (I don't think it does)
b) Why not move this bundle under the `io.openliberty.jakarta` project?